qt/lib/include/verovio-qt/verovioqttoolkit.h

```diff
@@ -0,0 +1,207 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        veroviodoc.h
+// Author:      Jonathan Schluessler
+// Created:     2017
+// Copyright (c) Authors and others. All rights reserved.
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef __VEROVIOQTTOOLKIT_H__
+#define __VEROVIOQTTOOLKIT_H__
+
+//----------------------------------------------------------------------------
+
+#include <QObject>
+#include <QVector>
+
+//----------------------------------------------------------------------------
+
+#include "toolkit.h"
+
+namespace vrv_qt {
+
+//----------------------------------------------------------------------------
+// VerovioDoc
+//----------------------------------------------------------------------------
+
+/**
+ * This class is a wrapper around the Verovio Toolkit (vrv::Toolkit).
+ */
+class Toolkit : public QObject {
```

Comment:
One more comment: I think you could do even better with a multiple inheritance with something like:
`class Toolkit : public QObject, public vrv::Toolkit` and avoid member duplication. You will probably have to make some member of vrv::Toolkit protected (and not private)

Reply:
I've considered all but the last review comments. I prefer to keep vrvQt::Toolkit and vrv::Toolkit linked by composition instead of inheritance for following reasons:

* Mixing multiple domains (Qt-domain with Verovio-domain) by multiple inheritance is considered bad style. In this case it would also be very obvious because it would mix upper-case (Verovio) with lower-case (Qt) function names.
* I already removed all duplicated class members in the previous commit by referring to the Getter-functions of vrv::Toolkit.
* I'd need to overload most of the Setter-functions of vrv::Toolkit, which thus would have to be marked virtual. At the moment vrv::Toolkit does not (apart from destructor) have any virtual functions.

Regarding the squares when using Leipzig or Gootville font: The fonts are loaded from the fontDirPath, specified in main.qml:53. Maybe it works if you specify an absolute path. The filenames are hard-coded in toolkit.cpp and should be Bravura-1.204.otf, Leipzig-5.2.ttf and Gootville-1.2.otf. In this branch these files exist.